### PR TITLE
Docker Desktop 3.0.1 release notes

### PR DESCRIPTION
**Pull request by stephen-turner**

### CODE REVIEW

**Review by usha-mandya — APPROVED** [review]
LGTM

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop Community 3.0.1
+2020-12-11
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)
```
Comment (mat007) [inline docker-for-mac/release-notes.md:22]: Should this not be `https://desktop.docker.com/mac/stable/50773/Docker.dmg`?

Comment (stephen-turner) [reply docker-for-mac/release-notes.md:22]: No, we usually use the Hub page not the direct link for the newest release.

Comment (mat007) [reply docker-for-mac/release-notes.md:22]: I’m pretty sure we moved away from this a few months ago, but that’s harmless anyway.

**Pull request merged**